Library/Koala/Koala.swift

```diff
@@ -1769,14 +1774,14 @@ private func properties(category category: KsApi.Category) -> [String:AnyObject]
     .withAllValuesFrom(parentProperties.prefixedKeys("parent_"))
 }
 
-private func properties(shareContext shareContext: ShareContext,
-                                     loggedInUser: User?,
-                                     shareActivityType: String? = nil) -> [String:AnyObject] {
+private func properties(shareContext: ShareContext,
+                        loggedInUser: User?,
+                        shareActivityType: UIActivityType? = nil) -> [String:Any] {
```

Comment:
`UIActivityType` has been upgraded to a proper enum in swift 3 so i went ahead and updated this api to be a bit stronger.